modules/apps/29-fee/ibc_module.go

```diff
@@ -46,8 +46,14 @@ func (im IBCModule) OnChanOpenInit(
 			chanCap, counterparty, version)
 	}
 
+	if versionMetadata.FeeVersion == "" {
```

Comment:
is this necessary?

Reply:
Nope, indded it is not. I had this removed locally and forgot to push :)